wg-opcodes/summaries/current-status.md

```diff
@@ -0,0 +1,56 @@
+# Metadata
+
+Title:            Current Status of OpCodes Workgroup
+Workgroup:        OpCodes
+Document Version: 1.0
+
+# Document Purpose
+
+This document is intended to provide a single source of truth of the status of ongoing op codes
+discussions relative to the upcoming hardfork. Any discussions not related to the latest upcoming
+fork will be referenced and postponed until the next hardfork or removed entirely if the discussion
+is in a complete form and no spec is planned to be proposed.  Implementation details or specs of
+particular op codes is out of scope of this document.
+
+# Current Status of OpCodes
+
+## OpCodes agreed upon for the next upcoming hard fork (May 2018)
+
+May 2018 hard fork spec proposal (for reference): https://github.com/bitcoincashorg/spec/pull/53/files
+
+OP_AND
+ * Spec proposed: https://github.com/bitcoincashorg/spec/pull/54/files
+OP_BIN2NUM
+ * Spec proposed: https://github.com/bitcoincashorg/spec/pull/54/files
+OP_CAT
+ * Spec proposed: https://github.com/bitcoincashorg/spec/pull/54/files
+OP_DIV
+ * Spec proposed: https://github.com/bitcoincashorg/spec/pull/54/files
+OP_MOD
+ * Spec proposed: https://github.com/bitcoincashorg/spec/pull/54/files
+OP_NUM2BIN
+ * Spec proposed: https://github.com/bitcoincashorg/spec/pull/54/files
+OP_OR
+ * Spec proposed: https://github.com/bitcoincashorg/spec/pull/54/files
+OP_SPLIT
+ * Spec proposed: https://github.com/bitcoincashorg/spec/pull/54/files
+OP_XOR
+ * Spec proposed: https://github.com/bitcoincashorg/spec/pull/54/files
+
+## OpCodes Under Discussion or Previously Discussed
+
+OP_DATASIGVERIFY
+ * Spec and implementation proposed, but not agreed upon.  See: https://github.com/shadders/uahf-spec/pull/6/files
+OP_GROUP
+ * Original proposal: https://medium.com/@g.andrew.stone/bitcoin-scripting-applications-representative-tokens-ece42de81285
+ * BUIP 077: https://github.com/BitcoinUnlimited/BUIP/blob/cd0763223fb79b0388b7d307bf4b2acdb9fc9a56/077.mediawiki
+ * Requirements proposed: https://www.yours.org/content/on-representative-tokens--colored-coins--bb7a829b965c/
+ * Implementation proposed to continue discussion: https://github.com/gandrewstone/BitcoinUnlimited/commit/7b8d8b708d1b15d0e05d6ef7e6f0dc1799efe186#diff-799cde8829295bd34ad577deb16cc152
```

Comment:
This is the best link I could find.  Let me know if there is a better one.

Reply:
branch w/ consensus code change only:
https://github.com/gandrewstone/BitcoinUnlimited/commits/opgroup_consensus

branch containing more wide code changes including e.g. wallet update:
https://github.com/gandrewstone/BitcoinUnlimited/commits/opgroup_fullnode

Reply:
Do these links need to go in before we merge?

Reply:
Don't have a strong opinion, but the provided links seems to give a more comprehensive view of the changes needed, both at the consensus and wallet level.